src/utils/hwManager.js

```diff
@@ -22,20 +26,69 @@ const getAccountsFromDevice = async () => {
  * signSendTransaction - Function.
  * This function is used for sign a send transaction.
  */
-const signSendTransaction = () => {
-  // TODO implement logic for this function
-  signTransaction();
-  throw new Error('not umplemented');
+const signSendTransaction = async (account, data) => {
+  const transactionObject = createSendTX(
+    account.info.LSK.publicKey,
+    data.recipientId,
+    data.amount,
+    data.data,
+  );
+
+  const transaction = {
+    deviceId: account.hwInfo.deviceId,
+    index: account.hwInfo.derivationIndex,
+    tx: transactionObject,
+  };
+
+  const [error, signature] = await to(signTransaction(transaction));
+
+  if (error) throw new Error(error);
```

Comment:
Why `await to` to catch the error and then throw the error again? IMO it would be simpler not to catch the error here.

Reply:
fixed

Reply:
I don't see a change here. Also pushed?